entrypoint.sh

```diff
@@ -51,9 +61,9 @@ fi
 echo "Latest ${VERSION_NAME}=${LATEST_VERSION}"
 
 # Set outputs.
-echo "::set-output name=current::${CURRENT_VERSION}"
-echo "::set-output name=latest::${LATEST_VERSION}"
-echo "::set-output name=repo::${REPO}"
+set_output current "${CURRENT_VERSION}"
+set_output latest "${LATEST_VERSION}"
+set_output repo "${REPO}"
```

Comment:
NIP: let's use quotes for names:

```suggestion
set_output "current" "${CURRENT_VERSION}"
set_output "latest" "${LATEST_VERSION}"
set_output "repo" "${REPO}"
```